Transparenz

milafrerichs opened the pull request:
## Beschreibung

Balkendiagramm für die Transparenz der Waffenexporte
Avg Transparenz in %
Dazugehöriges Unterdiagramm mit Subscores in %
## Dazugehöriges Issue
#78
## Screenshot

![bildschirmfoto 2014-12-12 um 12 25 46](https://cloud.githubusercontent.com/assets/688980/5411116/331ddcd6-81fd-11e4-8f40-e9e0f2ca30bb.png)
## Änderungen
### Hinzugefügt

Vertikales Balkendiagramm
Subvisualisierung für Transparenz Daten
### Geändert

Reihenfolge beim erstellen der Graphen (Axen zuletzt)
### Entfernt

milafrerichs committed:
Add VerticalBarchart to chart classes

ability to show a bar chart vertically
use class methods for drawing sepcific elements of the graph and
override them in the vertical version

milafrerichs committed:
Draw the axis after the values are drawn

axis should be on top of values instead of behind for better readability

milafrerichs committed:
Use verticalbarchart for detail transparency scores

milafrerichs committed:
Add interaction for transparency chart

update the subscore chart for the seletected country

milafrerichs commented:
Vorschlag: Text in die Balken im unteren Diagramm
Kürzen der texte, Informationen wegnehmen, 
für die längeren: Arms Profileration, Other für den Langen

milafrerichs commented:
![bildschirmfoto 2014-12-13 um 17 49 17](https://cloud.githubusercontent.com/assets/688980/5424462/65fe0aa6-82f0-11e4-92bf-ad40e274c5af.png)

milafrerichs committed:
Update transparency sub viz

hide axis, add axis text inside barchart
move labels to the right
place both visualisations in one line

ClaudiaSchwegmann commented:
Ja, das ist gut. Ich würde die Reihenfolge in der kleineren Graphik änder, so dass "Other" ganz unten ist. Es soll ja auch auf Deutsch sein. Ich schlage folgende Übersetzung vor:
Öffentlicher Jahresbericht - wenn das zu lang ist: Berichterstattung. Hover over: "Häufigkeit und Aktualität staatlicher Waffenexportberichte"
Strategie. Hover over: "Informationen zur Waffenexportstrategie"
Genehmigungsanträge. Hover over: "Informationen zu von Art und Umfang der gestellten Waffenexportanträge"
Exportgenehmigung. Hover over: "Informationen zu von Art und Umfang der genehmigten Waffenexporte"
Zwischenhandel. Hover over: "Informationen zu Art und Umfang genehmigten Waffenhandels als Zwischenhändler."
Transitlizenzen: Hover over: "Informationen zu Art und Umfang der Transitlizenzen"
Exporte. Hover over: "Informationen zu Art und Umfang tatsächlicher Exporte."
Kontrolle: "Informationen über die Durchsetzung bestehender Regeln"
Waffenverbreitung. Hover over: Informationen zu Verhinderung ungewollter Verbreitung von Waffen"
Weiter Aspekte: Hover over: "Informationen zu Lizenzverweigerungen, Lizenzannullierung, Lizenzsuspensionen und Lizenzerweiterung"

ClaudiaSchwegmann commented:
Und dann am Ende: Durchschnitt

milafrerichs commented:
![entwicklungsbarometer_-_transparenz](https://cloud.githubusercontent.com/assets/688980/5451346/ae223098-850f-11e4-8cb7-972323d93631.png)

milafrerichs committed:
Rename indikatoren to use german translation

add another field for description, shown on mouseover

milafrerichs committed:
Add new class for Saferworld Sub Viz

new class for easier overwriting of mouseover and drawValue methods
add mouseover for showing of text description

milafrerichs committed:
Add description to saferworld sub viz

add pointer to barchart rect to indicate mouseover functionality

milafrerichs committed:
Merge branch 'master' into 78-transparency

Conflicts:
	source/data/security/transparenz/saferworld.csv

milafrerichs committed:
Update transparency country names

use translated country names

ClaudiaSchwegmann commented:
Der letzte Punkt sollte nur "Weitere Aspekte" beschriftet sein würde ich sagen. Das "Informationen zu Lizenzverweigerungen, etc...." sollte nur im Hover over zu sehen sein.

milafrerichs committed:
Merge branch 'master' into 78-transparency

Conflicts:
	source/stylesheets/pages/_sicherheit.sass
